kolibri_instant_schools_plugin/assets/src/views/side-nav/keen-menu-port/index.vue

```diff
@@ -0,0 +1,165 @@
+<template>
```

Comment:
I wonder if it might be worth to add these 2 keen components to the core-api so we don't have to clone it here too. Thoughts? @rtibbles @indirectlylit

Reply:
I would rather avoid making changes to core Kolibri for the sake of this plugin, so I would prefer this to be copied here for now.

Reply:
I don't think it _really_ matters ... I think better to just leave it for now since it's not something we're re-using outside the navbar